index.html

```diff
@@ -2275,11 +2271,6 @@ <h3>Processing Errors</h3>
     <section>
       <h2>Contexts and Vocabularies</h2>
 
-      <p class="issue" title="(AT RISK) Hash values might change during Candidate Recommendation">
-This section lists cryptographic hash values that might change during the
-Candidate Recommendation phase based on implementer feedback that requires
-the referenced files to be modified.
-      </p>
```

Comment:
We specifically do not want to remove this issue because it's something implementers need to be aware of. Replace this change with a `data-number` markup: https://respec.org/docs/#data-number

Reply:
Done.  @msporny, please re-review.